### ISSUE TITLE
Changing scroll speed and deactivating arrow notes

### ISSUE THREAD

**Issue by Droucked** [Artemis-chan/osu-DIVA#143]
Hello, i have 2 problems
First, i'm terrible at fast notes and the notes are extremly fast since i started playing this mode, so i wanted to change it but f3 and f4 doesnt seems to do anything, is there any way to change the scroll speed?
Second, I'm a project diva fan but i hate the arrow keys since i play with dfjk so with the arrow keys it would make it like 8k on osu!mania (which i suck), is there any way to deactivate these arrow notes and only leave that playstation buttons? also i've seen someone play a map that only has these 4 buttons which made me think its possible.

also ignore the fact that my account was created today, i just passed my whole life using Github without an account lol

**Comment by Artemis-chan** [comment]
Yeah, you can enable the **No Doubles** mod to disable the arrow altogether.
Also you should be able to slow down by enabling **Half Time**/**Daycore** mod, then change the speed under customize mod section.

**Comment by Droucked** [comment]
> Yeah, you can enable the **No Doubles** mod to disable the arrow altogether. Also you should be able to slow down by enabling **Half Time**/**Daycore** mod, then change the speed under customize mod section.

**Comment by Droucked** [comment]
> you should be able to slow down by enabling Half Time/Daycore mod, then change the speed under customize mod section.
So there is no way to decrease the scroll speed without enabling half time?

**Comment by Droucked** [comment]
> you should be able to slow down by enabling Half Time/Daycore mod, then change the speed under customize mod section.

So there is no way to decrease the scroll speed without half time mode?

**Comment by Artemis-chan** [comment]
No, not at the moment, might add later.
